src/extensions/starter_dashlet/types.ts

```diff
@@ -0,0 +1,15 @@
+import { IDiscoveredTool } from '../../types/IDiscoveredTool';
+import { IRunningTool } from '../../types/IState';
+import { IDiscoveryResult } from '../gamemode_management/types/IDiscoveryResult';
+import { IGameStored } from '../gamemode_management/types/IGameStored';
+export interface IConnectedProps {
```

Comment:
Not a huge fan of the naming here. We have a very generic interface name (IConnectedProps) in a super generic file (types.ts) that is intended for exactly one specific component. Makes the code hard to understand as there is no easy way to make the connection